README.md

```diff
@@ -10,3 +11,17 @@ uvicorn qr-generator:app --reload
 ```
 
 - Ensure [public](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/restricting-the-visibility-of-forwarded-ports#overview) port forwarding in codespace.
+
+## Running the Dockerfile
+
+```bash
+$ docker build -t my-fastapi-app .
+$ docker run -p 80:80 my-fastapi-app
+```
+
+## Access the website
+
+- In your browser, navigate to ```https://mbaoma-ominous-yodel-7pj9wxg69jx2w6j5-8000.preview.app.github.dev/v1/qr?url=<your-url>```
```

Comment:
I think doing localhost might be a better choice instead of the GitHub url. The one you have right now is specific to your user

Reply:
I agree.